data-reconciliation-app/README.md

```diff
@@ -49,13 +44,37 @@ A makefile provides a frontend to interacting with the project, this is used bot
 
 ```text
 help                 💬 This help message :)
-lint                 🌟 Lint & format, will not fix but sets exit code on error, use in CI
-lint-fix             🔍 Lint & format, try to fix & update code, run locally
 build                🔨 Build the Application
-test                 🧪 Run tests, used for local development
-start-host           🏃 Start the application on a test CCF network
-deploy               🏡 Deploy application to Azure mCCF
-destroy              💣 Destroy application
-e2e-test             🤖 Run end to end tests
-clean                🧹 Clean up local files
+test                 🧪 Test the Data Reconciliation Application in the sandbox
+test-docker-virtual  🧪 Test the Data Reconciliation Application in a Docker sandbox
+test-docker-enclave  🧪 Test the Data Reconciliation Application in a Docker enclave
+test-mccf            🧪 Test the Data Reconciliation Application in a Managed CCF environment
+start-host           🏃 Start the CCF network using Sandbox.sh
+clean                🧹 Clean the working folders created during build/demo
+build-virtual        📦 Build Virtual container image from Dockerfile
+build-enclave        📦 Build Enclave container image from Dockerfile
```

Comment:
nit: move build commands so they stay together

Reply:
Change it in the actual makefile